static/loadSave.js

```diff
@@ -42,25 +42,32 @@ function setLocalFile(name, file) {
 }
 
 function LoadSave() {
-    this.modal = $('#load-save');
+    this.modal = null;
     this.alertSystem = new Alert();
     this.alertSystem.prefixMessage = "Load-Saver: ";
     this.onLoad = _.identity;
     this.editorText = '';
     this.extension = '.txt';
-    this.modal.find('.local-file').change(_.bind(this.onLocalFile, this));
-
-    this.modal.find('.save-button').click(_.bind(this.onSaveToBrowserStorage, this));
-    this.modal.find('.save-file').click(_.bind(this.onSaveToFile, this));
 
     this.base = window.httpRoot;
     if (!this.base.endsWith('/')) {
         this.base += '/';
     }
 
     this.fetchBuiltins();
+    this.initializeIfNeeded();
```

Comment:
I think this is not needed here, and we can postpone the init until run is called on the editor

Reply:
correct, have changed it